### DISCUSSION TITLE
Is it possible to get this data programmatically in SPFX ?

### DISCUSSION THREAD

**Discussion by aadilwani** [SharePoint/sp-dev-docs#7633] [Q&A] [answered]
Is it possible to get this data programmatically in SPFX ?

![image](https://user-images.githubusercontent.com/17984143/148434434-db0c5e14-04d3-4602-8552-f3759c7df483.png)

**Comment by AJIXuMuK** [comment, marked as answer]
@aadilwani - unfortunately, no.
Currently Settings context menu is not available in SPFx API.
